source/blog/2018-10-12-the-ember-times-issue-68.md

```diff
@@ -52,8 +52,17 @@ responsive: true
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Module Unification is in the final stretch!](https://github.com/emberjs/ember.js/issues/16373)
+Have you heard of **Ember's future default project layout**? If you haven't, you can read about it the [Module Unification RFC](https://github.com/emberjs/rfcs/blob/master/text/0143-module-unification.md). 
+
+The initial work on Module Unification is **nearly complete**, as **[there are only a handful of tasks left](https://github.com/emberjs/ember.js/issues/16373)** and some of the tasks just need a good amount of spelunking time.
+If you can donate some of your time to help finish this up, the entire community will forever be in your debt 💖
+
+Also, if you are a user of namespaced components, [this RFC](https://github.com/emberjs/rfcs/pull/367) is a must read. Please leave your thoughts on the [Module Unification Packages RFC](https://github.com/emberjs/rfcs/pull/367) as nested components affect the majority of Ember projects.
+
+To learn more about Module Unification, feel free to join the #st-module-unification channel on the [Ember Discord](https://discordapp.com/invite/zT3asNS)!
```

Comment:
i recently learned that you can link to a Discord channel on the web. so could change to `[#st-module-unification](https://discordapp.com/channels/480462759797063690/484527343331704832)`

Reply:
silly native app

Reply:
should the invite link be kept? or no?

Reply:
i say keep both to encourage the discord to be moar active, cool?